docker/DOCKERHUB.md

```diff
@@ -0,0 +1,118 @@
+_Note: this is the source of truth for DHIS2 Docker images. Any changes to this document need to be
+manually synced to Dockerhub since we do not have a paid account._
```

Comment:
Maybe these lines can be commented out, so that they don't accidentally end up in the actual Dockerhub readme.

Like: 
```
[//]: # (Comment line)
```